jestSetup.ts

```diff
@@ -25,6 +25,29 @@ window.flushAllPromises = async () => {
 fetchMock.enableMocks()
 jest.mock('src/shared/utils/errors')
 
+/*
+  N.B. when using react testing library render()
+  the mocked values in src/external/parser are ignored.
+  So, need to mock here as well
+*/
+jest.mock('src/external/parser', () => ({
```

Comment:
need to mock this for all jest tests that render components

Reply:
thanks for pointing this out. I refactored this implementation to look like the other ones and was able to delete the other mocking instances...except for 1 where it was testing a specific response from the parser and needed to be mocked on its own